main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -221,7 +221,8 @@ object CompletionProvider {
       SignatureCompleter.getCompletions(context) ++
       FieldCompleter.getCompletions(context) ++
       OpCompleter.getCompletions(context) ++
-      MatchCompleter.getCompletions(context)
+      MatchCompleter.getCompletions(context) ++
+      EnumTagCompleter.getCompletions(context)
```

Comment:
Instead of adding the EnumTagCompleter here, add it to the syntactic context for Pat.